test/Telegram.Bot.Tests.Integ/Stickers/StickersTests.cs

```diff
@@ -568,14 +568,14 @@ public async Task Should_Set_First_Sticker_Keywords()
         Sticker firstSticker = stickerSet.Stickers.First();
 
         await BotClient.SetStickerKeywordsAsync(
-            sticker: firstSticker.FileId,
+            sticker: new InputFileId(firstSticker.FileId),
             keywords: keywords
         );
 
         await Task.Delay(1_000);
 
         await BotClient.SetStickerKeywordsAsync(
-            sticker: firstSticker.FileId,
+            sticker: new InputFileId(firstSticker.FileId),
```

Comment:
@tuscen should we change `FileId` type to `InputFileId` for  `FileBase.File` so we don't need a conversion here? or sould we reverse this change in *Request classes?

Reply:
We can add overloads that accept strings and call ctors and methods with `InputFileId` instead of strings.

Reply:
Having impilict casting operator in `InputFileId` would do the same but it diminishes the goal to have domain-specific value object here. Wizou suggested to get rid of `InputFIleId` and `InputFileUrl` altogether in favor of simple `string`, since these classes are our domain interpretation and not an API requirement.

Reply:
Yeah, it seems fair to consider the option to maybe remove these two types.

Reply:
No need to, you can leave it as it is. We’ll make a few changes later after the merge.

Reply:
Ok, thanks and good luck